build-tools/packages/version-tools/src/semver.ts

```diff
@@ -60,6 +60,9 @@ export function bumpRange(
 
 		case "internal": {
 			const constraintType = detectInternalVersionConstraintType(range);
+			if (constraintType === "exact") {
+				throw new Error(`Can't bump exact specification from ${range}`);
+			}
```

Comment:
I think this is OK, but I think it would also make sense to treat it like `bumpInternalVersion`. That is, if bumpRange is passed an exact version constraint, then it just bumps that version, returning the same range constraint it had before (exact) but bumped by the `bumpType`. That's the spiritual intent of the function.

But throwing is OK too especially if we don't have any such cases today. I'd rather just fail, then easier to find and fix if we need to.